src/modules/jcms_rest/src/Plugin/rest/resource/PingRestResource.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Drupal\jcms_rest\Plugin\rest\resource;
+
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Provides a ping for smoke tests to call.
+ *
+ * @RestResource(
+ *   id = "ping_rest_resource",
+ *   label = @Translation("Ping rest resource"),
+ *   uri_paths = {
+ *     "canonical" = "/ping"
+ *   }
+ * )
+ */
+class PingRestResource extends AbstractRestResourceBase {
```

Comment:
Probably, @nlisgo shall I use the @RestResource support or can I go for something simpler like https://www.drupal.org/docs/8/creating-custom-modules/adding-a-basic-controller ? In which module should this be added? Does it need a new one?